.github/workflows/integration-tests-sca-tier-0-1-lin.yml

```diff
@@ -0,0 +1,87 @@
+name: Integration tests for SCA on Linux - Tier 0 and 1
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-sca-tier-0-1-lin.yml"
+        - "src/wazuh_modules/wm_sca.c"
+        - "src/wazuh_modules/wm_sca.h"
+        - "src/config/wmodules-sca.c"
+        - "src/Makefile"
+        - "tests/integration/conftest.py"
+        - "tests/integration/test_sca/**"
+
+jobs:
+  build:
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      BRANCH_BASE: ${{ github.base_ref || inputs.base_branch }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version-file: ".github/workflows/.python-version-it"
+          architecture: x64
+      # Build wazuh server for linux.
```

Comment:
I think it's better to use wazuh agent since this module will not be part of the server in the future.

Reply:
As in the cases of `msgraph` and `syscollector` tests, the `agent` tests that require a connection with a manager will fail since the connection to the simulator is not working.